pkg/features/features.go

```diff
@@ -63,3 +64,11 @@ func VSphereInPlaceUpgradeEnabled() Feature {
 		IsActive: globalFeatures.isActiveForEnvVar(VSphereInPlaceEnvVar),
 	}
 }
+
+// APIServerExtraArgsValidationDisabled is the feature flag for disabling the API server extra args validation.
+func APIServerExtraArgsValidationDisabled() Feature {
```

Comment:
We will check out the skip validation flag?

Reply:
Yes